### PR TITLE
'data: JoshuaEdo'

### PR DESCRIPTION
**Pull request by joshuaedo**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2726"></a>

### CODE REVIEW

**Review by Pradumnasaraf — COMMENTED** [review]
Hey, 
Are you sure you don't want to add any links?

In case you want to add them, here is the docs - https://linkfree.eddiehub.io/docs/how-to-guides/links

**Review by Pradumnasaraf — DISMISSED** [review]
Also, please fix your JSON profile naming. The casing is wrong, It should be `joshuaedo.json`. 

If you need any help feel free to ask.

File: data/JoshuaEdo.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Joshua Edo",
+  "displayStatsPublic": false,
+  "type": "personal",
+  "bio": "Hi there! You can call me Joshua(Josh for short) and I'm a Junior Frontend Web developer",
+  "avatar": "https://github.com/joshuaedo.png",
```
Comment (dan-mba) [inline data/JoshuaEdo.json:6]: Remove the comma at the end of the line

**Review by dan-mba — CHANGES_REQUESTED** [review]
The case of the filename should match the case of your GitHub user id.

**Comment by EmmaDawsonDev** [comment]
Do you need any help with the requested changes @joshuaedo ?

**Comment by joshuaedo** [comment]
## Overview of comments & changes requested ##
- [x] [Addition of links](https://github.com/EddieHubCommunity/LinkFree/pull/2726#pullrequestreview-1237939471)
- [x]  [Fixing the JSON profile naming](https://github.com/EddieHubCommunity/LinkFree/pull/2726#pullrequestreview-1237942394)
- [x]  [Removal of comma](https://github.com/EddieHubCommunity/LinkFree/pull/2726#discussion_r1062830805)
- [x]  [Help with changes](https://github.com/EddieHubCommunity/LinkFree/pull/2726#issuecomment-1382929229)

## Breakdown of changes addressed ##
### 1. Addition of links (Line 8 - 24) @Pradumnasaraf  ###
```diff
+ "links": [
+    {
+       "name": "Follow me on Twitter",
+       "url": "https://twitter.com/Thegr8Joshua",
+       "icon": "FaTwitter"
+     },
+     {
+       "name": "Follow me on LinkedIn",
+       "url": "https://www.linkedin.com/in/joshuaedo",
+       "icon": "FaLinkedIn"
+     },
+      {
+       "name": "Follow me on GitHub",
+       "url": "https://github.com/joshuaedo",
+       "icon": "FaGitHub"
+     }
+   ]
```
### 2. Fixing the JSON profile naming @Pradumnasaraf  & @dan-mba ###
```diff
- JoshuaEdo.json
+ joshuaedo.json
```
### 3. Removal of comma (Line 6) @dan-mba   ###
```diff
- "avatar": "https://github.com/joshuaedo.png",
+ "avatar": "https://github.com/joshuaedo.png"
```
### 4. Help with changes @EmmaDawsonDev   ###
Thanks for offering but I've already addressed the changes. If you find anything else that you think requires improvement, I'd appreciate it if you requested it to be changed.

#### Sorry for taking so long to reply, I was busy with school exams.

**Review by dan-mba — CHANGES_REQUESTED** [review]
To fix the failing JSON check you need to:
1. Add a `,` at the end of line 6
2. Move the `}` on line 7 to after the links section ends on line 24

**Review by dan-mba — APPROVED** [review]
Looks good to me

**Review by EmmaDawsonDev — APPROVED** [review]
Looks good! Thanks for contributing!

**Pull request merged**